test/unit/core/util/math-ex.test.ts

```
@@ -136,18 +150,18 @@ function testLongFloorDiv(x: number, y: number, expected: number, shouldThrow: b
     if (shouldThrow) {
       expect.fail(`floorDiv(${x}, ${y}); expected an arithmetic error`);
     }
-  } catch (e: ArithmeticError | any) {
-    if (!(e instanceof ArithmeticError)) {
-      throw e;
+  } catch (error: ArithmeticError | any) {
+    if (!(error instanceof ArithmeticError)) {
+      throw error;
     }
 
     if (!shouldThrow) {
-      expect.fail(`floorDiv(${x}, ${y}); Unexpected exception: ${e}`);
+      expect.fail(`floorDiv(${x}, ${y}); Unexpected exception: ${error}`);
     }
   }
 }
 
-function testLongFloorMod(x: number, y: number, expected: number, shouldThrow: boolean) {
+function testLongFloorModule(x: number, y: number, expected: number, shouldThrow: boolean) {
```

This might need to revert and add a suppress error